src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -95,53 +125,81 @@ export class HaConfigLovelaceDashboards extends LitElement {
           template: (requireAdmin: boolean) =>
             requireAdmin
               ? html`
-                  <ha-icon icon="hass:check-circle-outline"></ha-icon>
+                  <ha-icon icon="hass:check"></ha-icon>
                 `
               : html`
                   -
                 `,
-        },
-        sidebar: {
+        };
+        columns.sidebar = {
           title: this.hass.localize(
             "ui.panel.config.lovelace.dashboards.picker.headers.sidebar"
           ),
           type: "icon",
           template: (sidebar) =>
             sidebar
               ? html`
-                  <ha-icon icon="hass:check-circle-outline"></ha-icon>
+                  <ha-icon icon="hass:check"></ha-icon>
                 `
               : html`
                   -
                 `,
-        },
+        };
+      }
+
+      const columns2: DataTableColumnContainer = {
         url_path: {
           title: "",
           type: "icon",
           filterable: true,
           template: (urlPath) =>
-            html`
-              <mwc-button .urlPath=${urlPath} @click=${this._navigate}
-                >${this.hass.localize(
-                  "ui.panel.config.lovelace.dashboards.picker.open"
-                )}</mwc-button
-              >
-            `,
+            narrow
+              ? html`
+                  <paper-icon-button
+                    icon="hass:open-in-new"
+                    .urlPath=${urlPath}
+                    @click=${this._navigate}
+                  ></paper-icon-button>
+                `
+              : html`
+                  <mwc-button .urlPath=${urlPath} @click=${this._navigate}
+                    >${this.hass.localize(
+                      "ui.panel.config.lovelace.dashboards.picker.open"
+                    )}</mwc-button
+                  >
+                `,
         },
       };
+
       return { ...columns, ...columns2 };
     }
   );
 
   private _getItems = memoize((dashboards: LovelaceDashboard[]) => {
-    return dashboards.map((dashboard) => {
-      return {
-        filename: "",
-        ...dashboard,
-        icon: dashboard.sidebar?.icon,
-        title: dashboard.sidebar?.title || dashboard.url_path,
-      };
-    });
+    const defaultMode = (this.hass.panels?.lovelace?.config as any)?.mode;
+    const isDefault =
+      !localStorage.defaultPage || localStorage.defaultPage === "lovelace";
+    return [
+      {
+        icon: "hass:view-dashboard",
+        title: this.hass.localize("panel.states"),
+        default: isDefault,
+        sidebar: isDefault,
+        require_admin: false,
+        url_path: "lovelace",
+        mode: defaultMode,
+        filename: defaultMode === "yaml" ? "ui-lovelace.yaml" : "",
+      },
+      ...dashboards.map((dashboard) => {
+        return {
+          filename: "",
+          ...dashboard,
+          icon: dashboard.sidebar?.icon,
+          title: dashboard.sidebar?.title || dashboard.url_path,
```

Comment:
We will always have a title now and a url path. `dashboard.title` and `dashboard.url_path`